tests/e2e/fixtures/INR.js

```diff
@@ -0,0 +1,51 @@
+import { regionsEnum } from "../data/enums";
+import CheckoutPage from "../pages/CheckoutPage";
+import { doBillDeskPayment } from "../paymentFlows/redirectShopper";
+const shopperData = require("../data/shopperData.json");
+
+let checkoutPage;
+
+fixture`INR`
+    .page(`https://${process.env.SFCC_HOSTNAME}/s/RefArch/home`)
+    .httpAuth({
+        username: process.env.SANDBOX_HTTP_AUTH_USERNAME,
+        password: process.env.SANDBOX_HTTP_AUTH_PASSWORD,
+    })
+    .beforeEach( async t => {
+        await t.maximizeWindow()
+        checkoutPage = new CheckoutPage();
+        await checkoutPage.goToCheckoutPageWithFullCart(regionsEnum.IN);
+        await checkoutPage.setShopperDetails(shopperData.IN);
+    });
+
+test('UPI Success', async t => {
+    await doBillDeskPayment("billdesk_upi", "Success");
```

Comment:
Would be nice to send a boolean `true` instead of "Success" and then determine the text value in the `billdeskSimulator()` function